packages/mesa.rb

```diff
@@ -28,7 +28,11 @@ class Mesa < Package
   depends_on 'libxshmfence'
   depends_on 'elfutils'
   depends_on 'llvm'
```

Comment:
I think you can reduce the dependencies here.  A LOT of these are already installed from other packages.

Reply:
Yeah. Which ones?

Reply:
Follow the breadcrumb trail.  I think you can whittle it down to:
```
  depends_on 'wayland'
  depends_on 'elfutils'
  depends_on 'llvm'
```
But please confirm.

Reply:
Nope. All it installs is `python27`.

Reply:
I have a (slow) script that does it with `cut` and `tac`, but it requires the package file. Let me try that.

Reply:
If you add `libdrm` to that, then it's fine.

Reply:
That makes sense since that was added recently.